infra/modules/azure_core_infra/_modules/custom_roles/roles.tf

```diff
@@ -0,0 +1,16 @@
+resource "azurerm_role_definition" "pagopa_apim_list_secrets" {
```

Comment:
I'd remove every reference to 'pagopa' as these are general purpose modules

Reply:
It should be evaluated better, because the name chosen must be the same for all subscriptions